tox.ini

```diff
@@ -4,13 +4,19 @@ envlist = py39, flake8, black
 [testenv]
 deps =
     pytest
+    responses
```

Comment:
Hva er forskjellen på denne og `requests-mock` som vi har brukt andre steder?

Reply:
Lurte på om `fastapi.testclient.TestClient` kanskje lekte litt bedre med `responses` enn `requests-mock`, uten å [måtte aktivere](https://github.com/encode/starlette/issues/818) `real_http`. Skal dog sies jeg ikke undersøkte så mye nærmere, og litt for å prøve noe annet.

Reply:
Ah! Satt akkurat med det problemet tidligere i uka faktisk... Ser ut som `responses` er mer populært også, 3.3k stjerner på GitHub vs 260.